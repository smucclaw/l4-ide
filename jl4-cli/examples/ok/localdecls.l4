x MEANS
  y + z
  WHERE
    y MEANS 2
    z MEANS 3

#CHECK x
#EVAL x

outer MEANS
  middle
  WHERE
    middle MEANS
      inner
      WHERE
        inner MEANS 42

#EVAL outer
