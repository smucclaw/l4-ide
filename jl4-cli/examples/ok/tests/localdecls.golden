Parsing successful
Typechecking successful
localdecls.l4:7:8-9:
  NUMBER

localdecls.l4:8:7-8:
  5

localdecls.l4:18:7-12:
  42

